index.html

```diff
@@ -6,5 +6,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>update doc</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+
+      <nav>
+        <h1>Hello world</h1>
+        <h2> hello<h2>
```

Comment:
h2 does not have a slash in closing tag!!

Reply:
thanks you